src/haxeLanguageServer/server/HaxeServer.hx

```diff
@@ -289,120 +238,47 @@ class HaxeServer {
 			return;
 		}
 
-		var haxeResponse = buffer.getContent();
-
-		// invalid compiler argument?
-		var invalidOptionRegex = ~/unknown option `(.*?)'./;
-		if (invalidOptionRegex.match(haxeResponse)) {
-			var option = invalidOptionRegex.matched(1);
-			context.sendShowMessage(Error, 'Invalid compiler argument \'$option\' detected. '
-				+ 'Please verify "haxe.displayConfigurations" and "haxe.displayServer.arguments".');
-			return;
-		}
-
-		context.sendShowMessage(Error,
-			"Haxe process has crashed 3 times, not attempting any more restarts. Please check the output channel for the full error.");
-		trace("\nError message from the compiler:\n");
-		trace(haxeResponse);
-	}
-
-	function onData(data:Buffer) {
-		buffer.append(data);
-		while (true) {
-			if (nextMessageLength == -1) {
-				var length = buffer.tryReadLength();
-				if (length == -1)
-					return;
-				nextMessageLength = length;
-			}
-			var msg = buffer.tryReadContent(nextMessageLength);
-			if (msg == null)
-				return;
-			nextMessageLength = -1;
-			if (currentRequest != null) {
-				var request = currentRequest;
-				currentRequest = null;
-				request.onData(msg);
-				updateRequestQueue();
-				checkQueue();
-			}
-		}
+		// var haxeResponse = buffer.getContent();
```

Comment:
We have to move this section elsewhere, but it's going to depend on our architecture.